File: FindWeather/FindWeather/src/Models/WeatherData.java
package Models;

public class WeatherData {
    private final String location;
    private final String source;
    private final double temperature;
    private final String conditions;

    public WeatherData(String location, String source, double temperature, String conditions) {
        this.location = location;
        this.source = source;
        this.temperature = temperature;
        this.conditions = conditions;
    }

    public String getLocation() {
        return location;
    }

    public String getSource() {
        return source;
    }

    public double getTemperature() {
        return temperature;
    }

    public String getConditions() {
        return conditions;
    }

}

File: FindWeather/FindWeatherMySolution/models/WeatherData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FindWeatherMySolution.models
{
    public class WeatherData
    {
        public Location Location { get; set; }
        public double Temperature { get; set; }
        public string Conditions { get; set; }
        public string Source { get; set; }
        public DateTime Timestamp { get; set; }
    }
}
